pandas/tests/frame/test_indexing.py

```diff
@@ -3849,6 +3849,16 @@ def test_assigning_ops(self):
         df.loc[2:3, "b"] = Categorical(["b", "b"], categories=["a", "b"])
         tm.assert_frame_equal(df, exp)
 
+    def test_setitem_single_row_categorical(self):
```

Comment:
can you move to pandas/tests/frame/indexing/test_indexing.py

Reply:
Merged master, and moved to  pandas/tests/frame/indexing/test_categorical.py